db/models/Events.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose')
+
+const categories = ['music', 'dining']
+const accessibilityOptions = ['braille-menu', 'wheelchair-friendly']
+
+const eventSchema = mongoose.Schema(
+  {
+    ownerId: { type: mongoose.Schema.Types.ObjectId, ref: 'User' },
+    name: { type: String, required: true },
+    category: { type: [String], enum: categories, required: true },
```

Comment:
Yeah, I may have been wrong in the schema, I expect you may want to have more than one category.